apps/web-client/src/lib/db/cr-sqlite/order-reconciliation.ts

```diff
@@ -40,7 +40,22 @@ ascending
  */
 export async function getAllReconciliationOrders(db: DB): Promise<ReconciliationOrder[]> {
 	const result = await db.execO<ReconciliationOrder>(
-		"SELECT id, supplier_order_ids, finalized, updatedAt, created FROM reconciliation_order ORDER BY id ASC;"
+		`SELECT id, supplier_order_ids, finalized, updatedAt, created FROM reconciliation_order
+			ORDER BY id ASC;`
+	);
+	return result;
+}
+/**
+ * Retrieves currently reconciling orders, ordered by ID
+ascending
+ * @param db
+ * @returns ReconciliationOrder array
+ */
+export async function getReconcilingOrders(db: DB): Promise<ReconciliationOrder[]> {
```

Comment:
I feel like this can be merged with the function above, by passing an extra param optional param `finalised: boolean`?  If the param is present, add the where clause in the query with 1 or 0; if its not, don't include it and return all orders...